code/game/objects/items/breakable.dm

```diff
@@ -0,0 +1,129 @@
+//Breakable items
+
+//TODO:
+//multiples of a single fragment
+//add damaged text into the general examine proc
+//check slimes
+//add thrown response
+//projectiles?
+//kicking and biting?
+//change "hit" to something else?
+//hit sounds
+//punching hurting hands and stuff?
+//use kicking code
+//generalize out of devices
+
+/obj/item/device
```

Comment:
```suggestion
/obj/item
```
/device is a silly subtype. If you're going to do this, do it on the base `/obj/item` type or even `/obj`.
In fact, ideally this would be a component.

Reply:
Could you please let me know what you mean by component here?

Reply:
A `/datum/component` that can be `add_component`ed to anything. It's really not easy to work with so an inheritance-based approach is fine too, if you don't want to do it that way.

Reply:
I generalized it to `/obj/items` for now. I'll consider the ramifications of smashing other types of `/obj`.